src/sentry/replays/post_process.py

```diff
@@ -70,6 +70,14 @@ def generate_normalized_output(
         item["urls"] = item.pop("urls_sorted", None)
 
         item.pop("isArchived")
+        item.pop("click_tag", None)
+        item.pop("click_id", None)
+        item.pop("click_class", None)
+        item.pop("click_role", None)
+        item.pop("click_testid", None)
+        item.pop("click_aria_label", None)
```

Comment:
should this be `aria_label` or just label?

blueprint says we're going to expose `replay_click.label` as the field name. We could be consistent to drop `aria` internally to the code too

Reply:
Corrected this to expose the field as `replay_click.label`.  However, internally we'll continue the aria_label naming.  The value is extracted from that HTML attribute and needs to retain its specificity.